catalog/spatial/wfs/spatial-wfs-common/pom.xml

```diff
@@ -21,8 +21,25 @@
     </parent>
     <artifactId>spatial-wfs-common</artifactId>
     <name>DDF :: Spatial :: WFS :: Common</name>
+    <packaging>bundle</packaging>
```

Comment:
❓ why did this need to be switched to a bundle?

Reply:
i think I can change it back